action-helm-tools/common.sh

```diff
@@ -139,6 +139,11 @@ add_helm_repos() {
     echo $QLIK_HELM_DEV_PASSWORD | helm registry login --username $QLIK_HELM_DEV_USERNAME --password-stdin https://$QLIK_HELM_DEV_REGISTRY
   fi
 
+  if [ -n "$TALEND_ARTIFACTORY_USER" ]; then
+    echo "helm repo add talend https://artifactory.datapwn.com/artifactory/tlnd-helm-prod --username $TALEND_ARTIFACTORY_USER --password $TALEND_ARTIFACTORY_PASS"
```

Comment:
Will Github obfuscate the credentials in the log?

Reply:
Yes, see
https://github.com/qlik-trial/qcswrap-tmc-ipaas-rt-bookkeeper/actions/runs/5883007502/job/15954743713#step:7:100